### PR TITLE
feat: social hub doc page

### PR DESCRIPTION
**Pull request by fernando-cortez**
#### **WARNING**: Please make your PR against the `develop` branch.
- - -
**Select the type of change:**

- [ ] Small Changes - Typos, formatting, slight revisions
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**

This PR adds a landing page for the newly released Distributed Authority Social Hub Sample.

**Issue Number:**

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=1373) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=1373) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=1373) it.</sub>

**Review by Vic-Cooper — APPROVED** [review]

File: docs/learn/bitesize/bitesize-introduction.md [hunk 1]
```diff
@@ -9,7 +9,8 @@ The Bitesize Samples repository provides a series of sample code as modules to u
 * [2D Space Shooter Sample](bitesize-spaceshooter.md) - Learn more about physics movement and status effects using Netcode `NetworkVariables` and `ObjectPooling`.
 * [Invaders Sample](bitesize-invaders.md) - Learn more about game flow, modes, unconventional movement networked, and a shared timer.
 * [Client Driven Sample](bitesize-clientdriven.md) - Learn more about Client driven movements, networked physics, spawning vs statically placed objects, object reparenting.
-* [Dynamic Addressables Network Prefabs](bitesize-dynamicprefabs.md) - Learn more about the dynamic prefab system, which allows us to add new spawnable prefabs at runtime.
+* [Dynamic Addressables Network Prefabs](bitesize-dynamicprefabs.md) - Learn more about the dynamic prefab system, which allows you to add new spawnable prefabs at runtime.
+* [Multiplayer Distributed Authority Social Hub sample](bitesize-socialhub.md) - Learn how distributed authority changes the ownership of GameObjects between the clients and host.
```
[the remark this answers is not in the document]
Comment (fernando-cortez) [reply docs/learn/bitesize/bitesize-introduction.md:13]: @hershbansal99 If you wanted to add input too.

[the remark this answers is not in the document]
Comment (Vic-Cooper) [reply docs/learn/bitesize/bitesize-introduction.md:13]: Happy with that, thanks for your suggestion!

Does the multiplayer game have to be co-operative though? Does it not work the same with competitive games?

[the remark this answers is not in the document]
Comment (Vic-Cooper) [reply docs/learn/bitesize/bitesize-introduction.md:13]: Actually I think the phrase at the end might be redundant - the context for this use case is implicitly in a multiplayer game. 
Unless it really only does work in a co-operative game in which case we can call that out.

[the remark this answers is not in the document]
Comment (fernando-cortez) [reply docs/learn/bitesize/bitesize-introduction.md]: Right, we could just leave it as learning about host migration and ownership transfer and end the sentence there.

You wouldn't design a competitive game with distributed authority, but you're right, there may be other archetypes of multiplayer games. DA is not necessarily used for just co-operative games.

File: docs/learn/bitesize/bitesize-introduction.md [hunk 2]
```diff
@@ -9,7 +9,10 @@ The Bitesize Samples repository provides a series of sample code as modules to u
 * [2D Space Shooter Sample](bitesize-spaceshooter.md) - Learn more about physics movement and status effects using Netcode `NetworkVariables` and `ObjectPooling`.
 * [Invaders Sample](bitesize-invaders.md) - Learn more about game flow, modes, unconventional movement networked, and a shared timer.
 * [Client Driven Sample](bitesize-clientdriven.md) - Learn more about Client driven movements, networked physics, spawning vs statically placed objects, object reparenting.
-* [Dynamic Addressables Network Prefabs](bitesize-dynamicprefabs.md) - Learn more about the dynamic prefab system, which allows us to add new spawnable prefabs at runtime.
+* [Dynamic Addressables Network Prefabs](bitesize-dynamicprefabs.md) - Learn more about the dynamic prefab system, which allows you to add new spawnable prefabs at runtime.
+* [Distributed Authority Social Hub sample](bitesize-socialhub.md) - Learn how to use features of distributed authority like host migration and NetworkObject ownership transfer.
```
Comment (fernando-cortez) [inline docs/learn/bitesize/bitesize-introduction.md:13]: Just for consistency to either remove `Sample` or add it to each sample bullet.

File: docs/terms-concepts/distributed-authority.md [hunk 3]
```diff
@@ -50,3 +50,4 @@ For more information about how distributed authority works in Netcode for GameOb
 - [Race conditions](../basics/race-conditions.md)
 - [Spawning synchronization](../basics/spawning-synchronization.md)
 - [Deferred despawning](../basics/deferred-despawning.md)
+- [Multiplayer Distributed Authority Social Hub sample](../learn/bitesize/bitesize-socialhub.md)
```
Comment (fernando-cortez) [inline docs/terms-concepts/distributed-authority.md:53]: Shall we drop Multiplayer here?

**Pull request merged**